README.md

```diff
@@ -1,8 +1,47 @@
 # elr143 News Server
 
-Please add the following files to connect to the databases:
+You can find the hosted server here: https://elr143-news-server.onrender.com
 
-.env.development -> this should contain PGDATABASE=nc_news
+This is an API which accesses application data programmatically. It mimics the backend functionality of a site such as Reddit and will provide information to the front end architecture.
+
+## Please add the following files to connect to the databases:
 
+.env.development -> this should contain PGDATABASE=nc_news
 .env.test -> this should contain PGDATABASE=nc_news_test
 
+### Setup Instructions:
+
+1. In your terminal, navigate to your chosen directory, then copy and paste the following:
+`git clone https://github.com/ELR143/elr143-news-server.git`
+
```

Comment:
If you're telling them to clone, it's also worth mentioning that they'll have to then navigate into the folder/repo with `cd elr143-news-server`